src/NServiceBus.AcceptanceTests/Routing/when_replying_to_a_message_sent_to_specific_instance.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace NServiceBus.AcceptanceTests.ScaleOut
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NServiceBus.Routing;
+    using NUnit.Framework;
+
+    [Explicit("We don't yet have a way to inspect on which receiver a message has been received.")]
+    public class When_replying_to_a_message_sent_to_specific_instance : NServiceBusAcceptanceTest
+    {
+        const string ReceiverEndpoint = "ReplyingToAMessageSentToSpecificInstance.Receiver";
+
+        [Test]
+        public async Task Reply_address_should_be_automatically_set_to_that_specific_instance()
```

Comment:
>  You can set the reply address to a specific instance "manually" by calling ReplyOptions.RouteReplyToThisInstance

but then it should state that action somewhere in the test name. I still don't think the automatically adds any valuable information to the test.